src/cmake/templates/osconfig.spec.in

```diff
@@ -69,9 +70,12 @@ fi
 %systemd_post osconfig.service
 systemctl enable osconfig.service
 systemctl start osconfig.service
+systemctl enable osconfig-platform.service
+systemctl start osconfig-platform.service
 
```

Comment:
Don't do this. Increase the attack surface. Don't enable and don't start osconfig-platform at install time. 

Can do it and won't break anything except making a root service run unnecessarily that exposes a HTTP server increases the attack surface w/o reason.